libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java

```diff
@@ -52,12 +52,9 @@ public void onReceive(final Context context, final Intent intent) {
 
   MapboxNavigationNotification(Context context, MapboxNavigation mapboxNavigation) {
     this.mapboxNavigation = mapboxNavigation;
-    Locale locale = mapboxNavigation.options().locale();
-    if (locale == null) {
-      locale = LocaleUtils.getDeviceLocale(context);
-    }
+    RouteOptions routeOptions = mapboxNavigation.getRoute().routeOptions();
```

Comment:
The notification shouldn't be initialized without a valid route, as [the route is set](https://github.com/mapbox/mapbox-navigation-android/blob/master/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java#L393-L406) before the service is started (and the service initializes the notification). 

The route options should also always have a voice unit / language if the user has fetched the route with `NavigationRoute` as we added the functionality for defaults with the passing of the `Context`.  _But_, we should add those null checks in `DistanceUtils` like @Guardiola31337 mentioned because we cannot guarantee someone has fetched the route with `NavigationRoute`.